Sunrise/ViewModels/CartViewModel.swift

```diff
@@ -0,0 +1,92 @@
+//
+// Copyright (c) 2016 Commercetools. All rights reserved.
+//
+
+import ReactiveCocoa
+import Result
+import ObjectMapper
+import Commercetools
+
+class CartViewModel: BaseViewModel {
+
+    // Inputs
+    let refreshObserver: Observer<Void, NoError>
+
+    // Outputs
+    let isLoading: MutableProperty<Bool>
+    let numberOfItems: MutableProperty<String>
+
+    let cart: MutableProperty<Cart?>
+
+    // MARK: Lifecycle
+
+    override init() {
+        isLoading = MutableProperty(false)
+        let (refreshSignal, observer) = Signal<Void, NoError>.pipe()
+        refreshObserver = observer
+
+        cart = MutableProperty(nil)
+        numberOfItems = MutableProperty("")
+        numberOfItems <~ cart.producer.map { cart in String(cart?.lineItems?.count ?? 0) }
+
+        super.init()
+
+        refreshSignal
+        .observeNext { [weak self] in
+            self?.queryForActiveCart()
+        }
+    }
+
+    // MARK: - Data Source
+
+    func numberOfItemsInSection(section: Int) -> Int {
+        return cart.value?.lineItems?.count ?? 0
+    }
+
+    func itemNameAtIndexPath(indexPath: NSIndexPath) -> String {
+        return cart.value?.lineItems?[indexPath.row].name?.localizedString ?? ""
+    }
+
+    func itemSkuAtIndexPath(indexPath: NSIndexPath) -> String {
+        return cart.value?.lineItems?[indexPath.row].variant?.sku ?? ""
+    }
+
+    func itemSizeAtIndexPath(indexPath: NSIndexPath) -> String {
+        return cart.value?.lineItems?[indexPath.row].variant?.attributes?.filter({ $0.name == "size" }).first?.value as? String ?? "N/A"
+    }
+
+    func itemImageUrlAtIndexPath(indexPath: NSIndexPath) -> String {
+        return cart.value?.lineItems?[indexPath.row].variant?.images?.first?.url ?? ""
+    }
+
+    func itemPriceAtIndexPath(indexPath: NSIndexPath) -> String {
+        return cart.value?.lineItems?[indexPath.row].price?.value?.description ?? "N/A"
+    }
+
+    func itemTotalPriceAtIndexPath(indexPath: NSIndexPath) -> String {
```

Comment:
I would also keep here the `lineItemTotal...` name. When I saw `item` I was not sure what that was, but if named `lineItem` the association is immediate. Line item is something very specific in e-commerce, while "item" is something generic.